### PR TITLE
Multiline strings

### PR DESCRIPTION
**Pull request by joakim-hove**
Use raw string literals for test decks.

### CODE REVIEW

**Comment by joakim-hove** [comment]
jenkins build this please

File: tests/parser/ScheduleTests.cpp [hunk 1]
```diff
@@ -623,55 +625,55 @@ BOOST_AUTO_TEST_CASE(TestCrossFlowHandling) {
...
+ 10  JUL 2007 /
+ 10  AUG 2007 /
+/
+COMPDAT
+ 'OP_1'  9  9   3  9 'OPEN' 1*   32.948   0.311  3047.839 1*  1*  'X'  22.100 /
+/
+WELOPEN
+ 'OP_1' SHUT /
+ '*'    OPEN 0 0 3 /
+ 'OP_2' SHUT 0 0 0 4 6 / \n
```
Comment (bska) [inline tests/parser/ScheduleTests.cpp:659]: Lingering `\n` here.

File: tests/parser/ScheduleTests.cpp [hunk 2]
```diff
@@ -623,55 +625,55 @@ BOOST_AUTO_TEST_CASE(TestCrossFlowHandling) {
...
+ 'OP_3' SHUT 0 0 0 /
+/
+DATES             -- 4
+ 10  JUL 2008 /
+/
+WELOPEN
+ 'OP_1' OPEN /
+ 'OP_2' OPEN 0 0 0 4 6 / \n
```
Comment (bska) [inline tests/parser/ScheduleTests.cpp:667]: Lingering `\n` here.

File: tests/parser/ScheduleTests.cpp [hunk 3]
```diff
@@ -623,55 +625,55 @@ BOOST_AUTO_TEST_CASE(TestCrossFlowHandling) {
...
+ 'OP_3' OPEN 0 0 0 /
+/
+DATES             -- 5
+ 10  OKT 2008 /
+/
+WELOPEN
+ 'OP_1' SHUT 0 0 0 0 0 / \n
```
Comment (bska) [inline tests/parser/ScheduleTests.cpp:674]: Lingering `\n` here.

File: tests/parser/ScheduleTests.cpp [hunk 4]
```diff
@@ -721,40 +723,40 @@ BOOST_AUTO_TEST_CASE(CreateScheduleDeckWellsAndConnectionDataWithWELOPEN) {
...
+ 10  JUL 2008 /
+/
+WELOPEN
+ 'OP_1' OPEN /
+/
+DATES             -- 3
+ 10  OKT 2008 /
+/
+WELOPEN
+ 'OP_1' SHUT 0 0 0 0 0 / \n "
```
Comment (bska) [inline tests/parser/ScheduleTests.cpp:751]: Lingering `\n` here.

File: tests/parser/ScheduleTests.cpp [hunk 5]
```diff
@@ -721,40 +723,40 @@ BOOST_AUTO_TEST_CASE(CreateScheduleDeckWellsAndConnectionDataWithWELOPEN) {
...
+/
+DATES             -- 4
+ 10  NOV 2008 /
+/
+WELOPEN
+ 'OP_1' OPEN / \n "
```
Comment (bska) [inline tests/parser/ScheduleTests.cpp:757]: Lingering `\n` here.

File: tests/parser/ScheduleTests.cpp [hunk 6]
```diff
@@ -764,51 +766,52 @@ BOOST_AUTO_TEST_CASE(CreateScheduleDeckWithWELOPEN_TryToOpenWellWithShutCompleti
...
+ 10  JUL 2008 /
+/
+WELOPEN
+ 'OP_1' OPEN /
+/
+DATES             -- 3
+ 10  OKT 2008 /
+/
+WELOPEN
+ 'OP_1' SHUT 0 0 0 0 0 / \n "
```
Comment (bska) [inline tests/parser/ScheduleTests.cpp:794]: Lingering `\n` here.

File: tests/parser/ScheduleTests.cpp [hunk 7]
```diff
@@ -764,51 +766,52 @@ BOOST_AUTO_TEST_CASE(CreateScheduleDeckWithWELOPEN_TryToOpenWellWithShutCompleti
...
+/
+COMPDAT
+ 'OP_1'  9  9   1   1 'OPEN' 1*   32.948   0.311  3047.839 1*  1*  'X'  22.100 /
+/
+DATES             -- 4
+ 10  NOV 2008 /
+/
+WELOPEN
+ 'OP_1' SHUT 0 0 0 0 0 / \n "
```
Comment (bska) [inline tests/parser/ScheduleTests.cpp:803]: Lingering `\n` here.

**Review by bska — COMMENTED** [review]
Looks good except for 7 lingering instances of `\n` in the file.  Once we fix those this is ready to merge.

**Comment by joakim-hove** [comment]
Thanks @bska - you are noe thorogh reviewer!

**Comment by joakim-hove** [comment]
jenkins build this please

**Pull request merged**